simiyu1/JavaDevsNairobi
# 157354404 add ui tests

#### What does this PR do?
 - Add instrumentation tests

#### Description of Task to be completed?
 - create MainActivityUITests class

#### How should this be manually tested?
 - On navigation to the Androidtests directory and you should see the MainActivityUITests file
 - Additionaly, when you clone this repo, you can run the tests using the command `.gradlew tests`

#### What are the relevant pivotal tracker stories?
 - [#157354404](https://www.pivotaltracker.com/story/show/157354404)
